Lombiq.Tests.UI/Extensions/NavigationUITestContextExtensions.cs

```diff
@@ -307,4 +299,7 @@ public static void DismissAlert(this UITestContext context)
 
     public static Task GoToContentItemEditorByIdAsync(this UITestContext context, string contentItemId) =>
         context.GoToAdminRelativeUrlAsync($"/Contents/ContentItems/{contentItemId}/Edit");
+
+    private static UriBuilder ReturnRelativeUri(this UITestContext context, string relativeUrl) =>
```

Comment:
More like `GetAdminRelativeUrl`.

Reply:
I removed it as in the end it can be done with GetAbosuleUri